data/mapcss-mapping.csv

```diff
@@ -1073,7 +1073,7 @@ craft|electronics_repair;958;
 shop|appliance;959;
 amenity|casino;960;
 amenity|brothel;961;
-place|neighbourhood;[place=neighbourhood],[place=quarter];;name;int_name;962;
+place|quarter;[place=quarter];;name;int_name;962;
```

Comment:
This is not good. Keep old value for "neighbourhood", add new type somewhere else.

Reply:
Originally I thought like this too!

But the current OM's 962 feature type contain a mixture of quarters and neighbourhoods in all old map files.

This PR will make quarters looking more prominent, while neighbourhoods less (smaller font and 1 zoom level later).

So if we make type 962 for place=neighbourhood, then this less prominent style would apply to both quarters and neighbourhoods in all old maps, and hence all of them will look less prominent and will be less useful, which is not nice.

The way its done (type 962 for place=quarter) will just increase font sizes a little bit, but won't touch the zoom level. I think this is a much better option, i.e. not breaking for the old maps. :+1:

Reply:
Originally I thought like this too!

But the current OM's 962 feature type contain a mixture of quarters and neighbourhoods in all old map files.

This PR will make quarters looking more prominent, while neighbourhoods less (smaller font and 1 zoom level later).

So if we make type 962 for place=neighbourhood, then this less prominent style would apply to both quarters and neighbourhoods in all old maps, and hence all of them will look less prominent and will be less useful, which is not nice.

The way its done (type 962 for place=quarter) will just increase font sizes a little bit, but won't touch the zoom level. I think this is a much better option, i.e. not breaking for the old maps. 👍